SpiffWorkflow/spiff/specs/service_task.py

```
@@ -19,8 +19,14 @@ def _result_variable(self, task):
         return f'spiff__{escaped_spec_name}_result'
 
     def _execute(self, task):
-        def evaluate(expression):
-            return task.workflow.script_engine.evaluate(task, repr(expression))
+        def evaluate(param):
+            try:
+                param['value'] = task.workflow.script_engine.evaluate(task,
+                        param['value'])
+            except:
+                pass
```

I worry about swallowing an error here - it may make debugging a problem with parameters very hard to track down.  If someone misspells a variable name, how will they now what was wrong?